src/main/java/uk/gov/justice/digital/hmpps/keyworker/services/KeyworkerStatsService.java

```diff
@@ -127,7 +132,15 @@ public PrisonKeyWorkerStatistic generatePrisonStats(String prisonId) {
                     .sortOrder(SortOrder.ASC)
                     .build();
             ResponseEntity<List<StaffLocationRoleDto>> activeKeyWorkers = nomisService.getActiveStaffKeyWorkersForPrison(prisonId, Optional.empty(), pagingAndSorting, true);
-            log.info("There are currently {} active key workers in {}", activeKeyWorkers.getBody().size(), prisonId);
+
```

Comment:
do we have 2 meanings for 'active' here

Reply:
Yes, active in NOMIS but not active in keyworker